This phylogeny shows evolution of SARS-like betacoronaviruses including samples from the novel coronavirus (nCoV) outbreak in Wuhan. Coronaviruses from the Wuhan outbreak are colored in red, while related SARS-like coronaviruses are colored in blue. [Click here to zoom into the nCoV outbreak clade](/groups/blab/sars-like-cov?clade=nCoV).

There were 6 SNPs present in the nCoV samples in the first and last few bases of the alignment that were masked as likely sequencing artifacts. Additionally, a SNP at 18460 in BetaCoV/Wuhan/IVDC-HB-04/2020 appears directly adjacent to long stretch of ambiguous bases and so has also been masked. Additionally, sample BetaCoV/Wuhan/IVDC-HB-05/2019 may have sequencing artifacts that make it appear more diverged than it actually is. This sample has been dropped from the above analysis until resequencing occurs. Full details on bioinformatic processing can be found [here](https://github.com/blab/sars-like-cov).

The Wuhan coronavirus genomes were generously shared by scientists at the Shanghai Public Health Clinical Center & School of Public Health, Fudan University (sample *WH-Human_1*), at the National Institute for Viral Disease Control and Prevention, China CDC, Beijing, China (samples *BetaCoV/Wuhan/IVDC-HB-01/2019*, *BetaCoV/Wuhan/IVDC-HB-05/2019*, *BetaCoV/Wuhan/IVDC-HB-04/2020*) at the Institute of Pathogen Biology, Chinese Academy of Medical Sciences & Peking Union Medical College, Beijing, China (sample *BetaCoV/Wuhan/IPBCAMS-WH-01/2019*), and at the Wuhan Institute of Virology, Chinese Academy of Sciences, Wuhan, China (sample *BetaCoV/Wuhan/WIV04/2019*) via [GISAID](https://gisaid.org). We gratefully acknowledgement their contributions.

<div>
    <a href="https://gisaid.org">
        <img alt="gisaid-logo" width="120" src="https://www.gisaid.org/fileadmin/gisaid/img/schild.png"/>
    </a>
</div>
